### PR TITLE
terra-functional-testing with screenshots - Part 3/3

### PR DESCRIPTION
**Pull request by benbcai**
### Summary
Pull in visual regression in terra-functional-testing. Ensure tests use a unique screenshot name and regenerated all screenshots for the following components:

- terra-modal-manager
- terra-navigation-layout
- terra-navigation-side-menu
- terra-notification-dialog
- terra-popup
- terra-slide-group
- terra-slide-panel
- terra-slide-panel-manager
- terra-tabs
- terra-theme-provider
- terra-time-input

### Deployment Link

https://terra-framework-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

**Review by StephenEsser — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -87,14 +87,14 @@
     "pretest": "npm run lint",
     "postinstall": "link-parent-bin && npm run compile && npm run bootstrap:hoist",
     "start": "webpack-dev-server",
-    "start-heroku": "tt-serve-static --port $PORT --site './build' ",
-    "start-prod": "tt-serve --env.disableHotReloading -p",
-    "start-static": "npm run compile:prod && tt-serve-static --site ./build",
+    "start-heroku": "npm run terra -- express-server --port $PORT --site './build'",
```
Comment (mjhenkes) [inline package.json:90]: does this line work as `terra express-server --port   $PORT --site './build'`?

Comment (benbcai) [reply package.json]: Good catch! https://github.com/cerner/terra-framework/pull/1368/commits/8bca0bfcb0012fd846f1631ca5ddcd05740b59b5

**Review by mjhenkes — APPROVED** [review]

**Review by ryanthemanuel — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -87,14 +87,14 @@
...
+    "start-static": "npm run compile:prod && terra express-server --site ./build",
     "test": "npm run lint && npm run jest && npm run wdio",
-    "wdio-default": "wdio",
-    "wdio-lowlight": "NODE_OPTIONS=--max-old-space-size=2048 THEME=clinical-lowlight-theme wdio",
-    "wdio-fusion": "THEME=orion-fusion-theme wdio",
-    "wdio": "npm run wdio-default && npm run wdio-lowlight && npm run wdio-fusion"
+    "wdio-default": "terra wdio",
+    "wdio-lowlight": "NODE_OPTIONS=--max-old-space-size=2048 terra wdio --themes clinical-lowlight-theme",
+    "wdio-fusion": "terra wdio --themes orion-fusion-theme",
+    "wdio": "terra wdio --keepAliveSeleniumDockerService && NODE_OPTIONS=--max-old-space-size=2048 terra wdio --themes clinical-lowlight-theme --keepAliveSeleniumDockerService && terra wdio --themes orion-fusion-theme"
```
Comment (yuderekyu) [inline package.json:97]: i'm curious, with the theme context strategy, do we still need to increase the node memory via the node_options env variable?

**Review by yuderekyu — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (benbcai) [reply package.json:97]: Oops, I didn't see this comment before merging. I'm not sure...this is to match the existing `wdio-lowlight` script.

**Pull request merged**